fe/src/App.js

```diff
@@ -1,21 +1,21 @@
 import "./index.css";
```

Comment:
- change this to either `Home` or `Welcome` 
![image](https://user-images.githubusercontent.com/73863685/206156191-83c73f82-ebd8-4b50-87fd-e3b06b4bdf79.png)